conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureCollections.java

```diff
@@ -45,6 +45,20 @@ public static <T> void addAll(Collection<T> addTo, Iterable<? extends T> element
         }
     }
 
+    @SuppressWarnings("unchecked")
+    public static <T> void addAllNonNull(Collection<T> addTo, Iterable<? extends T> elementsToAdd) {
+        Preconditions.checkNotNull(elementsToAdd, "elementsToAdd cannot be null");
+        if (elementsToAdd instanceof Collection) {
+            // This special-casing allows us to take advantage of the more performant
+            // ArrayList#addAll method which does a single System.arraycopy.
+            addTo.addAll((Collection<T>) elementsToAdd);
```

Comment:
Doesn't this allow null elements to be added?

Reply:
Yes. And so does the `Map#putAll`